CefGlue.Common.Shared/Serialization/State/TypeMethodInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Xilium.CefGlue.Common.Shared.Serialization.State
+{
+    internal record TypeMethodInfo
```

Comment:
records should be pure DTOs
```suggestion
    internal class TypeMethodInfo
```